### ISSUE TITLE
Update crosslinks for GI Bill Comparison Tool

### ISSUE THREAD

**Issue by aprocik1** [department-of-veterans-affairs/va.gov-team#99481]
## Description
The product team is updating the GI Bill Comparison Tool to include this data:

- GI Bill school information from [this modern tool](https://www.va.gov/education/gi-bill-comparison-tool/) (VA.gov)
- Yellow Ribbon school information from [this modern tool](https://www.va.gov/education/yellow-ribbon-participating-schools/) (VA.gov)
- Reimbursement rates for national tests from [this legacy system](https://inquiry.vba.va.gov/weamspub/buildSearchNE.do) (WEAMS)
- Reimbursement rates for licenses and certifications from [this legacy system](https://inquiry.vba.va.gov/weamspub/buildSearchCountryLCCriteria.do) (WEAMS)

As a part of this update, the product team is updating the tool's URL and IA is updating the tool's placement on VA.gov. As a result, we need to update the tool's crosslinks.

First, we need to write a new, one-sentence link description to account for the new data that's being added to the tool. The description will appear on these crosslinks:

- Education and training mega menu fly out, promo spot
- [VA education and training benefits](https://www.va.gov/education/) page, promo spot
- [VA education and training benefits](https://www.va.gov/education/) page, "More information and resources" section

Then, we need to update these crosslinks in Drupal: 

- [VA education and training benefits](https://www.va.gov/education/) page, "More information and resources" section
- [Licensing and certification tests and prep courses](https://www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/licensing-and-certification-tests/) page, "What types of licensing and certification tests will my benefits cover?" section and "Tips for using the tool" section
- [National tests and prep courses](https://www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/national-tests/) page, "Approved admissions tests for pre-health professional programs" section

### Step 1: Update description
- [x] DB: Draft updated link description
- [ ] BP: Review

### Step 2: Update crosslinks
- [x] DB: Audit VA.gov for crosslinks
- [ ] DB: Draft updated crosslinks in Drupal
- [ ] ST: Copyedit and publish

## Criteria
- [ ] All crosslinks to the GI Bill Comparison Tool have been updated on VA.gov

**Comment by d-beyer** [comment]
@aprocik1 Here are two options for the link description:
- Compare VA education benefits at different schools, and find reimbursement rates for tests, licenses, and certifications.
- Compare VA education benefits at schools and training providers, and check reimbursement rates for tests, licenses, and certifications.

**Comment by aprocik1** [comment]
@d-beyer, thanks! Here's my recommendation:

> Learn about VA education benefits at different schools, and check reimbursement rates for licensing, certification, and national tests.

Here are some notes about my recommendation:

- "Learn about" is broader than "compare," which seems most appropriate, since the Tool includes lost of information about benefits and schools, beyond just comparing them.
- On other rates pages, we use "find," "get," and "review." Personally, "check" seems most appropriate to me.
- I think we only reimburse for "tests" -- regardless of if those are for licensing and certification or national.

Feel free to accept / reject my recommendation as you see fit. Then, start a thread with Beth about this in Slack. She wants to be involved in drafting this.

**Comment by d-beyer** [comment]
@aprocik1 Following new information and context, I went back to the drawing board. I wanted to run it by you again before proposing it. Here's the new revision:

> Compare VA education benefits at schools and employers, and find covered national, licensing, and certification tests.

My rationale:
- "Compare" vs. "learn about": Compare is a stronger verb, and I think more accurate, even if it is less explicit about the fact that you can search for other information about schools. If you can compare information, it is sufficiently implied that you can find information. Our benefit detail pages handle the "learn about" part, and "search for and compare" would be overkill.
- Added "employers": This should be captured.
- "Find" vs. "check": Encompasses both finding available tests and viewing their reimbursement amounts.
- "Covered" over "reimbursement/reimbursable": Reduces the amount of jargon. I imagine this is the part you're most likely to push back on. I think "covered" tells the beneficiary everything they need to know, but I can understand it might be less consistent.
- Reordered test types: Increases clarity that these are all adjectives describing "tests", improves flow.

**Comment by aprocik1** [comment]
@d-beyer, thanks for sharing this clear and thoughtful analysis. I have one suggestion--you can take it or leave it. 

If you're not familiar with VA's test approval process, the phrase "find covered...tests" may be confusing. I would recommend something like "find national, licensing, and certification tests that VA [covers / has approved / reimburses]."

**Comment by d-beyer** [comment]
The final version of the short description I drafted was:

> Compare VA education benefits at schools and employers, and find national, licensing, and certification tests covered by VA.

I completed an [audit of links that point to the GI Bill Comparison Tool, WEAMS, and the Yellow Ribbon Tool](https://dvagov.sharepoint.com/:x:/r/sites/SitewideCAIA/Shared%20Documents/2.0%20Product%20work/GI%20Bill%20Comparison%20Tool/GI%20Bill%20comp%20tool%20cross-links%20tracker.xlsx?d=we722e973b452419dabeb5eb4294bbd58&csf=1&web=1&e=H1pQ3E). (This was done manually since the link audit function doesn't work for non-drupal pages.) These updates can be made once the tool updates happen in March.

**Comment by bethpottsVADEPO** [comment]
@mnorthuis FYI, I talked with SMEs today about 1/ the abbreviated messaging for the promo spot and 2/ the longer teaser text for descriptions accompanying links on the hub landing page and elsewhere. 

**Promo spot:**
They felt that it wasn't necessary to mention everything in the text for the promo spot, so we might go with this: 

Use this tool to compare your GI Bill benefits by school.

**Teaser text:**
They didn't think we needed to call out Yellow Ribbon in the teaser text. Noting here that we're trying to be careful not to talk about this as "approved schools" but rather as "schools with approved programs." This is mainly because not every program at a school will be approved. Here's what we worked on (we may tweak further, but this is where we landed today): 

Compare your GI Bill benefits at schools with approved programs. And find out how much we’ll pay you back for licensing and certification tests and national tests. You can also search for employers with approved training programs for on-the-job training and apprenticeships.

**Comment by mnorthuis** [comment]
@bethpottsVADEPO now that we are planning to remove the teaser text from the hub pages, we won't need this for that page (sorry).  It can, however, be used as context for other places that we crosslink to the GIBCT.

**Comment by bethpottsVADEPO** [comment]
@mnorthuis Right! There are plenty of other places, like related links, where we'll use this text. What's the timing on removing teaser text from the hub pages?

**Comment by mnorthuis** [comment]
Just opened ticket yesterday - https://github.com/department-of-veterans-affairs/va.gov-team/issues/106037
